articles/queries.md

```diff
@@ -32,6 +32,9 @@ How to create a query:
 
 4. Select **Save**, enter a name and description for your query, select the frequency that the query should run at, and select **Save query**.
 
+## Targeting hosts using labels
+
```

Comment:
I think we should clarify this is a premium feature
```suggestion

_Available in Fleet Premium_

```

Reply:
👍  good catch